src/utils/random-user.tsx

```diff
@@ -0,0 +1,11 @@
+const baseUrl = 'https://randomuser.me/api/?seed=motivation&inc=picture,name';
+
+export const getUsers = async (
```

Comment:
Since it's called `getUsers`, I would expect it to return the array of users. Cannot we just return it and handle them in any way you need outside of this function?

Reply:
got it and solved